libraries/AC_PID/AC_PID.cpp

```diff
@@ -252,7 +252,7 @@ float AC_PID::update_all(float target, float measurement, float dt, bool limit,
     P_out *= _pid_info.Dmod;
     D_out *= _pid_info.Dmod;
 
-    // boost output if required
+    // scale output if required
     P_out *= boost;
     D_out *= boost;
```

Comment:
Might as well change the variable name too?

Reply:
Fixed